### ISSUE TITLE
[REQUEST] Freecam AntiCheat

### ISSUE THREAD

**Issue by Jujumatic** [jiangdashao/Matrix-Issues#912]
**What features/checks do you need?:**
An anti freecam would be great to add as it seems there's no checks currently done for it. I think it would be good to stop people snooping around the map and especially the underground map, it would be a good thing to have even though they already can't interact with chests or mobs so they can't see the map and sneak into players' secret buildings and whatnot

**Comment by Olexorus** [comment]
I don't think this is possible. Since Freecam hacks are usually entirely client sided, they're undetectable if the cheat is coded properly (the same was as for example esp hacks can't be detected).
